javascript/embedded-login/.env.example

```diff
@@ -1,7 +1,6 @@
-AM_URL=$AM_URL
+SERVER_URL=$SERVER_URL
 REALM_PATH=$REALM_PATH
 SCOPE=$SCOPE
 TIMEOUT=$TIMEOUT
```

Comment:
Might not need "TIMEOUT" any more, if we're hardcoding it in `main.js` (unless it's used elsewhere?)